source/NVDAObjects/window/_msOfficeChart.py

```diff
@@ -0,0 +1,972 @@
+# -*- coding: UTF-8 -*-
+#NVDAObjects/window/_msOfficeChartConstants.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2014-2015 James Teh, Michael Curren, NV Access Limited
```

Comment:
Fix the copyright statement.  Something like Copyright (C) 2014-2017 NV access Limited, Dinesh Kaushal